MdePkg/Include/Library/DebugLib.h

```diff
@@ -51,6 +51,8 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #define DEBUG_CACHE     0x00200000  // Memory range cachability changes
 #define DEBUG_VERBOSE   0x00400000  // Detailed debug messages that may
                                     // significantly impact boot performance
+#define DEBUG_MM_SUPV   0x00800000  // MU_CHANGE: This is separately defined to allow repetetive

```

Comment:
I was expecting the mm entrypoint prints to be added under this flag as well. But on the other hand, it may just qualify a verbose flag.

Reply:
Alright. The bit can of course represent anything within its scope over time (i.e., be added to new MM Supervisor debug prints), so I want to make sure that the scope of "MM Supervisor" is suitable at this time.

Main concern is that "MM Supervisor" is a large scope and "syscall dispatch" messages are a small/specific piece of that so everything else in the MM Supervisor would need to be covered under pre-existing bits (like info, warn, error, verbose, etc) leaving this to represent the syscall messages for it to be useful in containing the message output.

If we go with this approach, I think this is one change that really needs to go into TianoCore first to avoid a bit assignment conflict and ensure it matches the upstream without conflict.

Reply:
Synced offline, and determined that this is not the proper to proceed and could cause unnecessary ripples when upstreaming to the edk2. Will change the supervisor code to cover this issue instead.